Compiled from "CallbackException.java"
public class org.hibernate.CallbackException extends org.hibernate.HibernateException {
  public org.hibernate.CallbackException(java.lang.Exception);
    Code:
       0: aload_0
       1: ldc           #1                  // String An exception occurred in a callback
       3: aload_1
       4: invokespecial #2                  // Method "<init>":(Ljava/lang/String;Ljava/lang/Exception;)V
       7: return

  public org.hibernate.CallbackException(java.lang.String);
    Code:
       0: aload_0
       1: aload_1
       2: invokespecial #3                  // Method org/hibernate/HibernateException."<init>":(Ljava/lang/String;)V
       5: return

  public org.hibernate.CallbackException(java.lang.String, java.lang.Exception);
    Code:
       0: aload_0
       1: aload_1
       2: aload_2
       3: invokespecial #4                  // Method org/hibernate/HibernateException."<init>":(Ljava/lang/String;Ljava/lang/Throwable;)V
       6: return
}
